ptimeit/_impl.py

```diff
@@ -1,17 +1,25 @@
 import functools
 import time
 from contextlib import contextmanager
+from typing import Callable
 
 
 _depth = -1
 
 
 @contextmanager
-def timeit_section(section_name: str):
+def timeit_section(
+    section_name: str,
+    should_print: Callable[[float], bool] = lambda t: True,
```

Comment:
maybe we can just name this `cond` or `condition` ?

Reply:
Yeah: https://github.com/narekgharibyan/ptimeit/pull/1/commits/3b0982515d4e9a8b7f2ad5fa736a70b687c688a2